tslearn/bases.py

```diff
@@ -0,0 +1,217 @@
+from abc import ABCMeta, abstractmethod
+from sklearn.base import BaseEstimator
+from . import hdftools
```

Comment:
Up to now, we use absolute imports (we should maybe change it at some point)

Reply:
Using relative imports is better practice.

Reply:
I know but my point is that we should change all imports at once, one day. But OK, this point is probably not important / counter-productive.